rpm2.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
 
   s.files         = `git ls-files`.split("\n")
   s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
-  s.executables   = `git ls-files -- bin/*`.split("\n").map { |f| File.basename(f) }
+  s.executables   = `git ls-files -- exe/*`.split("\n").map { |f| File.basename(f) }
```

Comment:
There's neither a bin nor exe directory before this, so this is a no-op packaging-wise